### PR TITLE
Fixed some docs

### PR DESCRIPTION
**Pull request by KostyaTretyak**

### CODE REVIEW

**Comment by samwilson** [comment]
Looks good!

File: classes/Kohana/HTTP/Cache.php [hunk 1]
```diff
@@ -36,7 +36,7 @@ class Kohana_HTTP_Cache {
 	 *          )
 	 *      );
 	 *
-	 * @uses    [Cache]
+	 * @uses    Cache
```
Comment (shadowhand) [inline classes/Kohana/HTTP/Cache.php:39]: we're losing linking here

Comment (KostyaTretyak) [inline classes/Kohana/HTTP/Cache.php:39]: Here you can see a live example
http://ko.docs.org.ua/guide-api/Kohana_HTTP_Cache#factory

**Comment by acoulton** [comment]
FYI: this was merged directly into 3.3/master - I've merged to 3.3/develop.

**Pull request merged**